vc/consent-credential-v1.jsonld

```diff
@@ -9,11 +9,37 @@
       "ConsentCredential":"ex:ConsentCredential",
       "issuerDIDDoc":{
          "@id":"ex:issuerDIDDoc",
-         "@type":"@json"
+         "@context":{
```

Comment:
@rolsonquadras can you post an example of what an instance of `issuerDIDDoc` would look like?

Reply:
@llorllale Here you go.

```
"issuerDIDDoc":{
         "doc":{
            "@context":[
               "https://w3id.org/did/v1"
            ],
            "publicKey":[
               {
                  "controller":"did:example:123456789abcdefghi",
                  "id":"did:example:123456789abcdefghi#keys-1",
                  "publicKeyBase58":"H3C2AVvLMv6gmMNam3uVAjZpfkcJCwDwnZn6z3wXmqPV",
                  "type":"Secp256k1VerificationKey2018"
               },
               {
                  "controller":"did:example:123456789abcdefghi",
                  "id":"did:example:123456789abcdefghi#keys-2",
                  "publicKeyBase58":"H3C2AVvLMv6gmMNam3uVAjZpfkcJCwDwnZn6z3wXmqPV",
                  "type":"Ed25519VerificationKey2018"
               },
               {
                  "controller":"did:example:123456789abcdefghw",
                  "id":"did:example:123456789abcdefghw#key2",
                  "publicKeyBase58":"H3C2AVvLMv6gmMNam3uVAjZpfkcJCwDwnZn6z3wXmqPV",
                  "type":"RsaVerificationKey2018"
               }
            ],
            "service":[
               {
                  "id":"",
                  "priority":0,
                  "recipientKeys":[
                     "did:example:123456789abcdefghi#keys-2"
                  ],
                  "routingKeys":[
                     "76HmFbj8sds7jjdnZ4hMVcQgtUYZpEN1HEmPnCrH2Bby"
                  ],
                  "serviceEndpoint":"https://localhost:8090",
                  "type":"did-communication"
               },
               {
                  "id":"",
                  "priority":1,
                  "recipientKeys":[
                     "did:example:123456789abcdefghi#keys-1"
                  ],
                  "routingKeys":null,
                  "serviceEndpoint":"https://localhost:8090",
                  "type":"did-communication"
               }
            ]
         },
         "id":"did:peer:123456789abcdefghi#inbox"
      }
```